lib/puppet/provider/package/pip.rb

```diff
@@ -2,162 +2,123 @@
 # <http://pip.pypa.io/>
 
 require 'puppet/provider/package'
+require 'puppet/provider/package_targetable'
```

Comment:
@tkishel can you double check that we got the correct funtionality

Reply:
Looks good! 6.0.x introduces `PUP-5985 Don't use features for windows?` and this mergup keeps that and maintains the targetable functionality. I double-checked this file ... and all the others in `(PUP-1082) targetable package command`.